### ISSUE TITLE
(Suggestion) Make "Advanced Search" title more descriptive

### ISSUE THREAD

**Issue by gooyers** [ualbertalib/discovery#729]
This suggestion also came out of a PSC meeting on 11/18/2015. 

When discussing the fact that advanced search does not search EDS results, it was suggested that the title "Advanced Search" is misleading on the main advanced search page: https://search.library.ualberta.ca/advanced. 

Just as the "Articles Advanced Search" button at the top is intended to describe the function of that search, the "Advanced Search" should be named similarly. A specific name was not suggested, but to be accurate...it would need to be "Books, media, journals, databases, and more Advanced Search". 

That is certainly too wordy, but the core of the suggestion stands. That calling Blacklight Advanced Search simply "advanced search" does not suggest that the Advanced Search is limited to only the Blacklight index.

**Comment by redlibrarian** [comment]
Thanks @gooyers . As usual, the hardest problem is naming things. @ualbertalib/discovery-testing any thoughts on this one?

**Comment by linds37** [comment]
I'd like to suggest that we keep it simple and use Books Advanced Search
Databases & Journals will be included as well, but putting Journals in the name will make it sound like an article search and Databases, just to confusing. 

I still like best the idea of searching everything (including EDS) with the Advanced search, and it bringing back a Bento box display.

**Comment by seanluyk** [comment]
I think using the word advanced might be a little misleading itself. What we really have here is the ability to pre-facet. I wonder if we could bring this item for discussion with the larger group?

We could also call it the Books, Media, and More advanced search, with 'more' being our miscellaneous bucket

**Comment by redlibrarian** [comment]
This hasn't been reported as an issue at all this term. I'd like to close this issue and see if the issue arises in usability testing or problems reported by users.

**Comment by theLinkResolver** [comment]
I think having the "articles advanced search" button right under the title "Advanced search" helps a bit. The text in the search tips still needs revision ("local content", specifically), for which I opened discussion in #862. Maybe @ualbertalib/discovery-testing could set their sights on that issue and we can close this one.

**Comment by linds37** [comment]
As long as the Advanced Search doesn't search the same set of resources as the simple search, then it is still an issue. The ideal will be for it to search EDS as well. 
Until it does, I think it should be explicitly stated that it's an Advanced Search for Books, Media & More (I know it also searches Journals & Databases, but there is no way to state that succinctly).
